application/clicommands/VersionCommand.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/* Icinga Web 2 | (c) 2019 Icinga Development Team | GPLv2+ */
+
+namespace Icinga\Clicommands;
+
+use Icinga\Application\Version;
+use Icinga\Application\Icinga;
+use Icinga\Cli\Loader;
+use Icinga\Cli\Command;
+
+/**
+ * Shows version for Icinga Web 2 and modules
+ *
+ * The version command shows version numbers for Icinga Web 2, loaded modules and also PHP.
+ *
+ * Usage: icingacli --version
+ */
+class VersionCommand extends Command
+{
+    protected $defaultActionName = 'show';
+
+    /**
+     * Shows version for Icinga Web 2 and modules
+     *
+     * The version command shows version numbers for Icinga Web 2, PHP and the loaded modules.
+     *
+     * Usage: icingacli --version
+     */
+    public function showAction()
+    {
+        $getVersion = Version::get();
+        printf("%-12s  %-9s \n", 'Icinga Web 2', $getVersion['appVersion']);
+        printf("%-12s  %-9s \n", 'PHP Version', PHP_VERSION);
+
+        //Git Commit if there is one
+        if (isset($getVersion['gitCommitID'])) {
```

Comment:
If there is a Git commit ID, it should be shown beneath the Icinga Web 2 version.